File: Python/1 - Iniciante/1118.py
# coding UTF-8
repetir = True
confirm = False
while repetir:
    pos = int(0)
    som = float(0)
    while pos != 2:
        a = float(input())
        if 0 <= a <= 10:
            som += a
            pos += 1
        else:
            print('nova invalida')
    print('media = {:.2f}'.format(som / 2))

    while True:
        print('novo calculo (1-sim 2-nao)')
        b = int(input())
        if b == 2:
            repetir = False
            break
        elif b == 1:
            repetir = True
            break

File: go/1118.go
package main

import (
	"fmt"
)

func main() {
	op := 1
	var n float64
	var soma float64

	for op != 2 {
		soma = 0.0
		if op == 1 {
			for i := 1; i <= 2; {
				fmt.Scan(&n)
				if n >= 0 && n <= 10 {
					soma += n
					i++
				} else {
					fmt.Println("nota invalida")
				}
			}
			media := soma / 2
			fmt.Printf("media = %.2f\n", media)
		}
		fmt.Println("novo calculo (1-sim 2-nao)")
		fmt.Scan(&op)
	}
}
